api/stats.py

```diff
@@ -0,0 +1,111 @@
+def load():
+	from pickle import load
+	stats_dict = {}
+	stats_db = open('api/stats.db', 'rb')
+	try:
+		stats_dict = load(stats_db)
+		return stats_dict
+	except:
+		return {}
+
+stats_dict = load()
+
+def clear(update, job_queue):
+	global stats_dict
+	stats_dict = {}
+	print("cleared")
+
+def stats_check(update, context):
+	global stats_dict
+	#from global_stats import global_stats_dict
+	from time import gmtime, strftime
+
+	msg = update.message
+	user = msg.from_user.id
+	chat_id = update.message.chat_id
+	user_object = msg.from_user
+
+	increment(stats_dict, chat_id, user_object)
+	#increment(global_stats_dict, chat_id, user_object)
+
+	from pickle import dump
+	stats_db = open('api/stats.db', 'wb')
+	dump(stats_dict, stats_db)
+	stats_db.close()
```

Comment:
Can use a context manager here
```suggestion
	with open('api/stats.db', 'wb') as stats_db:
	        dump(stats_dict, stats_db)
```